src/main/java/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,19 @@
+package converter;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> data) {
```

Comment:
Don't begin method implementation with an empty line. Please remove the redundant empty line at the start of the `converterToTransaction` method.